ui/constants/publish_types.js

```diff
@@ -0,0 +1,2 @@
+export const FILE = 'File';
```

Comment:
Possibly Upload now as label, @seanyesmunt? (fine with file as internal name)

Reply:
IMO "file" makes more sense, the publish form already shows as a main title "Upload" for every mode.
The way I see it, the publish mode indicates what content is the user going to "upload".
"Post" is a special case, because is an actual file that will be "uploaded" but it creates the illusion that there is no file just like a regular web / blog post. 🧙 

( But that's just my opinion 🙃 )

Reply:
Not sure but I think I am leaning towards `File` and agree with @btzr-io

Reply:
makes sense, I can add it right now if you want

Reply:
still that won't fix the current overwrite / editing ux from the publish form